### ISSUE TITLE
Exits 0 on non-existent config file

### ISSUE THREAD

**Issue by janosh** [igorshubovych/markdownlint-cli#227]
When specifying a false config file location, `markdownlint` currently exits with status 0. This is unexpected behavior, especially when used as a pre-commit hook where it passes silently. You only see the warning below when running in verbose mode (`-v`).

```sh
$ pre-commit run markdownlint -v
markdownlint.............................................................Passed
- hook id: markdownlint
- duration: 0.22s

Cannot read or parse config file dotfiles/.markdownlint.yml: ENOENT: no such file or directory, open 'dotfiles/.markdownlint.yml'
```
